platform/android/MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/activity/maplayout/DebugModeActivity.java

```diff
@@ -9,20 +9,31 @@
 import android.util.Log;
 import android.view.MenuItem;
 import android.view.View;
+
 import com.mapbox.mapboxsdk.constants.Style;
 import com.mapbox.mapboxsdk.maps.MapView;
 import com.mapbox.mapboxsdk.maps.MapboxMap;
 import com.mapbox.mapboxsdk.maps.OnMapReadyCallback;
 import com.mapbox.mapboxsdk.testapp.R;
+import com.mapbox.mapboxsdk.testapp.model.constants.AppConstant;
 
-public class DebugModeActivity  extends AppCompatActivity {
+public class DebugModeActivity extends AppCompatActivity {
 
     private static final String TAG = "DebugModeActivity";
 
     private MapView mapView;
     private MapboxMap mapboxMap;
 
-    private String currentStyle = Style.MAPBOX_STREETS;
+    private int currentStyleIndex = 0;
+
+    private static final String[] STYLES = new String[]{
+            Style.getMapboxStreetsUrl(AppConstant.STYLE_VERSION),
+            Style.getLightStyleUrl(AppConstant.STYLE_VERSION),
+            Style.getDarkStyleUrl(AppConstant.STYLE_VERSION),
+            Style.getSatelliteStyleUrl(AppConstant.STYLE_VERSION),
+            Style.getSatelliteStreetsStyleUrl(AppConstant.STYLE_VERSION),
+            Style.getOutdoorsStyleUrl(AppConstant.STYLE_VERSION)
```

Comment:
Probably not a big deal, but I think the intention is to place Outdoors between Streets and Light in ordered lists.